### PR TITLE
Make sure that schemes use Xcode 8.3, add armv7s to architecture list…

### PR DESCRIPTION
**Pull request by TroubleMakerBen**
… for Puppet and Sasquatch, fix framework search path for puppet

### CODE REVIEW

File: MobileCenter/MobileCenter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -124,8 +124,8 @@
 		B21E29911E83521A00F9A22D /* MSUtility+StringFormatting.h in Headers */ = {isa = PBXBuildFile; fileRef = B21E298F1E83521A00F9A22D /* MSUtility+StringFormatting.h */; };
 		B21E29921E83521A00F9A22D /* MSUtility+StringFormatting.m in Sources */ = {isa = PBXBuildFile; fileRef = B21E29901E83521A00F9A22D /* MSUtility+StringFormatting.m */; };
 		B24F3F171D93A3FF00827213 /* MSLoggerTests.m in Sources */ = {isa = PBXBuildFile; fileRef = B24F3F161D93A3FF00827213 /* MSLoggerTests.m */; };
+		B289FF881F291F9D00AA2E06 /* MSDeviceInternal.h in Headers */ = {isa = PBXBuildFile; fileRef = B2C3C1831DB83A3E00CB83F7 /* MSDeviceInternal.h */; };
```
Comment (jaelim-ms) [inline MobileCenter/MobileCenter.xcodeproj/project.pbxproj]: Why is this changed?

Comment (jaelim-ms) [inline MobileCenter/MobileCenter.xcodeproj/project.pbxproj]: This is not a problem at all but it will add a pain when we merge. :cry:

Comment (TroubleMakerBen) [inline MobileCenter/MobileCenter.xcodeproj/project.pbxproj]: It was a duplicate entry.
